app/src/test/java/com/door43/usecases/CreateRepositoryTest.kt

```diff
@@ -52,6 +52,7 @@ class CreateRepositoryTest {
         every { targetTranslation.id }.returns("aa_gen_text_reg")
 
         every { resources.getString(R.string.pref_default_gogs_api) }.returns(apiUrl)
+        every { context.getString(R.string.gogs_user_agent) }.returns("btt-writer")
```

Comment:
Shouldn't these tests be looking for "btt-writer-android", not "btt-writer"?  It looks like we're setting it to "btt-writer-android" in app/src/main/res/values/strings_app_pref.xml.